Now x86_64-darwin fails:

```
============================= test session starts ==============================
platform darwin -- Python 3.10.8, pytest-7.1.3, pluggy-1.0.0
rootdir: /private/tmp/nix-build-python3.10-trustme-0.9.0.drv-0/trustme-0.9.0
collected 10 items / 1 error                                                   
==================================== ERRORS ====================================
____________________ ERROR collecting tests/test_trustme.py ____________________
ImportError while importing test module '/private/tmp/nix-build-python3.10-trustme-0.9.0.drv-0/trustme-0.9.0/tests/test_trustme.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/xm2lnwsm2mmnn2ijszybcbqikqlj3hlp-python3-3.10.8/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_trustme.py:20: in <module>
    import OpenSSL.SSL
E   ModuleNotFoundError: No module named 'OpenSSL'
=========================== short test summary info ============================
ERROR tests/test_trustme.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.18s ===============================
error: builder for '/nix/store/h27dgww1wnp1dnp4ap5fd15zaad7j2zl-python3.10-trustme-0.9.0.drv' failed with exit code 2;
```

--- a/pkgs/development/python-modules/trustme/default.nix
+++ b/pkgs/development/python-modules/trustme/default.nix
@@ -23,7 +23,7 @@ buildPythonPackage rec {
   checkInputs = [
     service-identity
     pytestCheckHook
-  ] ++ lib.optionals (!stdenv.isDarwin && !stdenv.isAarch64) [
+  ] ++ lib.optionals (!stdenv.isDarwin || !stdenv.isAarch64) [
     pyopenssl
   ];
 

